server/templates/home/searchresult.html

```diff
@@ -0,0 +1,185 @@
+                  <div class="input-group1 ">
+                    <h6>View</h6>
+                      <div class="input-group-prepend">
+                        
+                          <select class="view" name="search_type" >
+                              <option value="meta" selected>view1</option>
+                              <option value="elements">view2</option>
+                          </select>
+                      </div>
+                      <h6>Sort by</h6>
+                      <div class="input-group-prepend">
+                        
+                        <select class="sort-by" name="search_type">
+                            <option value="meta" selected>select1</option>
+                            <option value="elements">select2</option>
+                        </select>
+                      </div>
+                      <div class="input-group-append">
+                          <button class="submit-go" type="submit">Go</button>
+                      </div>
+                  </div>
+              </form>
+              </div>
+            </div>            
+            <hr>
+            <div class="card mb-3 border-0" >
+                <div class="row g-0">
+                  <div class="col-md-3">
+                    <img id="image"  src="{%static 'images/search1.jpg'%}" class="img-fluid" alt="search_image">
+                  </div>
+                  
+                  <div class="col-md-8" >
+                    <div class="card-body">
+                      <h5 class="card-title">manual</h5>
+                      <p class="content">how do i set up a BRC</p>
+                      <p>Contributor : <span>pradan</span></p>
+                      <p>Date: 2022</p>
+                      <p>Resource: <a href="#">View</a> | <a href="#">Download</a></p>
+
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <hr>
+
+              
+            <div class="card mb-3 border-0" >
+                <div class="row g-0">
+                  <div class="col-md-3">
+                    <img  src="{%static 'images/search2.jpg'%}" class="img-fluid card_img" alt="search_image" >
+                  </div>
+                  
+                  <div class="col-md-8">
+                    <div class="card-body">
+                      <h5 class="card-title">manual</h5>
+                      <p class="content">Natural Farming: A Guide</p>  
+                      <p>Contributor : <span>parambha</span></p>
+                      <p>Date: 2022</p>
+                      <p>Resource: <a href="#">View</a> | <a href="#">Download</a></p>
+
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <hr>
+        </div>
+        
+    </div>
+
+{%endblock content%}
+{% block script %}
+<script src="https://code.jquery.com/jquery-3.6.4.min.js"></script>
+
+
+
+<script>
+  $(document).ready(function() {
```

Comment:
Instead of populating the options using separate fetch, get them in the initial render using context. When a filter is changed rerender using get request with updated params